frontend/src/components/LinkButton/LinkButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import '../MyNavbar/MyNavbar.css'
+import {Button} from 'reactstrap';
+
+
+
+//Compact button component
+//Set prop 'target' to "_blank" to open the link in a new tab 
+const LinkButton = (props) => {
```

Comment:
Can you change this into a functional component please.

Reply:
Done